modules/core/shared/src/test/scala/gem/arb/ArbTime.scala

```diff
@@ -100,6 +94,9 @@ trait ArbTime {
   implicit val cogLocalDate: Cogen[LocalDate] =
     Cogen[(Int, Int)].contramap(d => (d.getYear, d.getDayOfYear))
 
+  implicit val cogDuration: Cogen[Duration] =
+    Cogen[(Long,Int)].contramap(d => (d.getSeconds, d.getNano))
+
```

Comment:
I'm still not entirely sure what a `Cogen` is. Does anyone have any references that can help me understand better?

Reply:
TL;DR if you have `Cogen[A]` and `Gen[B]` you get `Gen[A => B]`. The job of `Cogen[A]` is to take an `A` and absorb some information in order to perturb the random seed that's used to generate the return value `B`. This talk explains it very well. You can probably skip ahead to 36:00 if you already understand `Gen`.

    https://www.youtube.com/watch?v=UlZGZh7hfbs

(I'm off today but wanted to share that link.)